### PR TITLE
Add syncBulkActions config to avoid the large amount of metrics data written ES in a single sync bulk request.

### PR DESCRIPTION
**Pull request by Ax1an**
Please answer these questions before submitting a pull request

- Why submit this pull request?
- [ ] Bug fix
- [ ] New feature provided
- [x] Improve performance

- Related issues

___
### Bug fix
- Bug description.

- How to fix?

___
### New feature or improvement
- Describe the details and related test reports.

   Add syncBulkActions config to avoid the large amount of metrics data written ES in a single bulk request.

image
image

### CODE REVIEW

**Review by zifeihan — COMMENTED** [review]
good.

**Comment by wu-sheng** [comment]
Follow #5697 to update two change log files.

**Comment by wu-sheng** [comment]
If you plan to merge this for now.

File: oap-server/server-storage-plugin/storage-elasticsearch-plugin/src/main/java/org/apache/skywalking/oap/server/storage/plugin/elasticsearch/StorageModuleElasticsearchConfig.java [hunk 1]
```diff
@@ -52,6 +52,7 @@
     private int superDatasetIndexShardsFactor = 5;
     private int indexRefreshInterval = 2;
     private int bulkActions = 2000;
+    private int syncBulkActions = 10000;
```
Comment (wu-sheng) [inline oap-server/server-storage-plugin/storage-elasticsearch-plugin/src/main/java/org/apache/skywalking/oap/server/storage/plugin/elasticsearch/StorageModuleElasticsearchConfig.java:55]: Why put this in 10k? Is there any testing, which could prove this provides better performance?

Comment (Ax1an) [reply oap-server/server-storage-plugin/storage-elasticsearch-plugin/src/main/java/org/apache/skywalking/oap/server/storage/plugin/elasticsearch/StorageModuleElasticsearchConfig.java:55]: In our production environment, it's 100k. However, considering that most people try skywalking, the machine configuration is not so high, and this is a configurable item, so the initial value is set to 10k.

File: changes/changes-8.2.0.md [hunk 2]
```diff
@@ -87,6 +87,7 @@
 * Fix end time bug in the query process.
 * Fix `Exporter INCREMENT mode` is not working.
 * Fix an error when executing startup.bat when the log directory exists
+* Add syncBulkActions config to avoid the large amount of metrics data written ES in a single sync bulk request.
```
Comment (wu-sheng) [inline changes/changes-8.2.0.md:90]: I think this description is not correct, you just separate the config, can't avoid that thing

File: oap-server/server-storage-plugin/storage-elasticsearch-plugin/src/main/java/org/apache/skywalking/oap/server/storage/plugin/elasticsearch/StorageModuleElasticsearchConfig.java (hunk 1 above)
Comment (wu-sheng) [reply oap-server/server-storage-plugin/storage-elasticsearch-plugin/src/main/java/org/apache/skywalking/oap/server/storage/plugin/elasticsearch/StorageModuleElasticsearchConfig.java:55]: I am a little concern about the size, whether it is too big for a normal user.

File: changes/changes-8.2.0.md (hunk 2 above)
Comment (wu-sheng) [reply changes/changes-8.2.0.md:90]: Could change this to 

> Add syncBulkActions configuration to set up the batch size of the trace/log persistent.

Comment (Ax1an) [reply changes/changes-8.2.0.md]: Done.

**Review by wu-sheng — APPROVED** [review]

**Pull request merged**